ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/BasicCallbackHandler.java

```diff
@@ -44,13 +43,10 @@ public void handle(final Callback[] callbacks)
       throws IOException, UnsupportedCallbackException {
     for (final Callback callback : callbacks) {
       if (callback instanceof NameCallback) {
-        final NameCallback nc = (NameCallback)callback;
+        final NameCallback nc = (NameCallback) callback;
         nc.setName(username);
-      } else if (callback instanceof ObjectCallback) {
```

Comment:
This should not be removed. LoginModules use `ObjectCallback` for the password. Some fallback to `PasswordCallback` for old CallbackHandlers that don't support `ObjectCallback` but newer versions of some LoginModules don't fall back to `PasswordCallback` (e.g., [link](https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-jaas/src/main/java/org/eclipse/jetty/jaas/spi/LdapLoginModule.java)), so removing this breaks support.

Let's also add support for `ObjectCallback` back in the unit tests in `BasicCallbackHandlerTest.java`.

Feel free to add a comment into the code here so others looking at the code aren't similarly confused and tempted to remove `ObjectCallback` support?

Reply:
We can't add ObjectCallback as it's not a standard callback - it's a jetty extension

Reply:
Let's chat about this later - I'd rather not have an explicit dependency to Jetty in the code (I think it's ok to bundle jetty-jaas so users can use login modules though)